scripts/__fixtures__/mixed.tsx

```
@@ -0,0 +1,22 @@
+  const { t } = useTranslation();
+
+  return (
+    <div>
+      {/* Translated */}
+      <h1>{t('common.title')}</h1>
+
+      {/* Hardcoded */}
+      <p>This text is hardcoded</p>
+
+      {/* Translated */}
+      <button>{t('common.submit')}</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)